brainzutils/cache.py

```diff
@@ -193,9 +193,9 @@ def set_many(mapping, time=None, namespace=None, encode=True):
     """
     # TODO: Fix return value
     result = _r.mset(_prep_dict(mapping, namespace, encode))
-    if time:
+    if expirein:
         for key in _prep_keys_list(list(mapping.keys()), namespace):
-            _r.pexpire(key, time * 1000)
+            _r.pexpire(key, expirein * 1000)
```

Comment:
interestingly, pexpire uses milliseconds but `expire` will do seconds. we should change this, as we use seconds everywhere else anyway.

Reply:
we decided to continue using pexpire so that we could do fractions of seconds. However, we should cast these to ints before passing it in, otherwise it won't work